substrate-node/runtime/src/lib.rs

```diff
@@ -444,6 +453,35 @@ impl pallet_scheduler::Config for Runtime {
     type WeightInfo = ();
 }
 
+parameter_types! {
+	pub const CouncilMotionDuration: BlockNumber = 3 * MINUTES;
+	pub const CouncilMaxProposals: u32 = 100;
+	pub const CouncilMaxMembers: u32 = 100;
+}
+
+type CouncilCollective = pallet_collective::Instance1;
+impl pallet_collective::Config<CouncilCollective> for Runtime {
+	type Origin = Origin;
+	type Proposal = Call;
+	type Event = Event;
+	type MotionDuration = CouncilMotionDuration;
+	type MaxProposals = CouncilMaxProposals;
+	type MaxMembers = CouncilMaxMembers;
+	type DefaultVote = pallet_collective::PrimeDefaultVote;
+	type WeightInfo = ();
+}
+
+type EnsureRootOrHalfCouncil = EnsureOneOf<
+	AccountId,
+	EnsureRoot<AccountId>,
+	pallet_collective::EnsureProportionAtLeast<_1, _2, AccountId, CouncilCollective>
```

Comment:
The docs say at least 60% so half of the council is not right

Reply:
There is a number of votes you can add to a proposal, so no need to change that here

Reply:
Next to the fact that this leaves room for easy human error it also means that 50% of a council can take over the council completely.

Reply:
I need to investigate how this actually works because this is some code I came across in many other substrate projects

Reply:
```
EnsureProportionAtLeast<N, D, AccountId, I>
```

N would stand for nominator and D for denominator? So N=2 and D=3 should work?

Reply:
that's 66%, it was also on my todo list to see how the code worked

Reply:
Then it should be N=3 , D=5